test/libsolidity/SemanticTest.cpp

```diff
@@ -321,7 +383,11 @@ void SemanticTest::printSource(ostream& _stream, string const& _linePrefix, bool
 void SemanticTest::printUpdatedExpectations(ostream& _stream, string const&) const
 {
 	for (auto const& test: m_tests)
-		_stream << test.format("", true, false) << endl;
+		_stream << test.format(
+			"",
+			m_gasCostFailure ? TestFunctionCall::RenderMode::ExpectedValuesActualGas : TestFunctionCall::RenderMode::ActualValuesExpectedGas,
```

Comment:
I'd use `ActualValuesActualGas` here too:

```suggestion
			m_gasCostFailure ? TestFunctionCall::RenderMode::ActualValuesActualGas : TestFunctionCall::RenderMode::ActualValuesExpectedGas,
```

Reply:
Marking as resolved. See https://github.com/ethereum/solidity/pull/10863#discussion_r578517480